src/views/SideMenu.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <aside>
     <a :href="'/'" class="menu-brand has-text-centered">
-      <img src="@/assets/img/logo.svg" alt="Camptocamp.org" />
+      <img src="@/assets/img/logo-noel.png" alt="Camptocamp.org" />
```

Comment:
```suggestion
      <img src="@/assets/img/logo_noel.png" alt="Camptocamp.org" />
```
Works with this change!